Exposure/Asset/Response/Asset.swift

```diff
@@ -178,6 +182,7 @@ extension Asset: Codable {
         rating = try container.decodeIfPresent(Float.self, forKey: .rating)
         markers = try container.decodeIfPresent([Marker].self, forKey: .markers)
         userData = try container.decodeIfPresent(AssetUserData.self, forKey: .userData)
+        durationMillis = try container.decodeIfPresent(Int64.self, forKey: .duration)
```

Comment:
Shall we keep the asset keys as same as the Exposure `duration`  ? 
`medias` object already have a key called `durationMillis`, so this may add some confusion in future

Reply:
I agree, let's keep the names from the json. I'll change it to "duration".